packages/core/src/ui/InlineToolbar/index.ts

```diff
@@ -148,12 +153,38 @@ export class InlineToolbar {
 
   /**
    * Apply format with data formed in toolbar
-   * @param toolName - name of the inline tool, whose format would be applied
+   * @param nameOfTheTool - name of the inline tool, whose format would be applied
    */
-  public apply(toolName: InlineToolName): void {
+  public formData(nameOfTheTool: InlineToolName): void {
+    const elementWithOptions = this.#inlineToolAdapter.formatData(nameOfTheTool, (data: InlineToolFormatData): void => {
+      this.apply(nameOfTheTool, data);
+    });
+
     /**
-     * @todo pass to applyFormat inline tool data formed in toolbar
+     * Do not render toolbar data former if element with options is null
+     * It means, that tool does not need any data, and callback will be triggered in adapter
      */
-    this.#formattingAdapter.applyFormat(toolName, createInlineToolData({}));
+    if (elementWithOptions === null) {
+      return;
+    }
+
+    this.#dataFormElement = elementWithOptions.element;
+
+    if (this.#toolbar === undefined) {
+      throw new Error('InlineToolbar: can not show formDataElement without toolbar');
+    }
+
+    this.#holder.appendChild(this.#dataFormElement);
   };
+
+  /**
+   * lalala
+   * @param toolName lalala
+   * @param formatData ;alala
+   */
+  public apply(toolName: InlineToolName, formatData: InlineToolFormatData): void {
```

Comment:
should not close actions, it should be optional

Reply:
this could be handeled by tool in future, since it renderes this element